spec.bs

```diff
@@ -721,32 +774,72 @@ To <dfn>create a request</dfn> given a [=URL=] |url|, a [=string=] <dfn for="cre
 1. Return |request|.
 
 <div algorithm>
+To <dfn>validate fetching response</dfn> given a [=response=] |response| and |responseBody|, and a

```

Comment:
What is the type of |responseBody|?

Reply:
It's "null, failure, or a byte sequence" according to the definition of [processResponseConsumeBody](https://fetch.spec.whatwg.org/#process-response-end-of-body).  I guess since it can be one of three things with different types, it doesn't get an explicit type declaration.